src/endpoints/sources.js

```diff
@@ -391,6 +403,29 @@ async function sendAny(req, res, url, accept) {
 }
 
 
+/**
+ * Generate an Express endpoint that receives a group overview
+ * determined by the request.
+ *
+ * @param {sourceExtractor} sourceExtractor - Function to provide the Source instance from the request
+ * @returns {expressEndpointAsync}
+ */
+function receiveGroupOverview(sourceExtractor) {
+  return async (req, res) => {
+    const groupSource = sourceExtractor(req);
+
+    authz.assertAuthorized(req.user, authz.actions.Read, groupSource);
```

Comment:
`authz.actions.Write`, not read!

Reply:
Oops, fixed in 9168b758615c8163486b6c8a6cee2e9415116b32.